html2asketch/model/symbolMaster.js

```diff
@@ -1,7 +1,13 @@
-import {generateID} from '../helpers/utils';
+import {
+  generateID,
+  getGroupLayout,
+  SMART_LAYOUT,
+} from '../helpers/utils';
 import Base from './base';
 import SymbolInstance from './symbolInstance';
 
+export const SYMBOL_SMART_LAYOUT = SMART_LAYOUT;
```

Comment:
I am actually not sure about this export here. I thinks it's quite intuitive to import it from utils. but it's just a detail.